src/components/Calculator.js

```diff
@@ -1,45 +1,69 @@
-import PropTypes from 'prop-types';
+import { useState } from 'react';
+import DigitsButtons from './logic/DigitsButtons';
+import OperatorsButtons from './logic/OperatorsButtons';
 import './Calculator.css';
 
-// Child Component
-function Button({ label }) {
-  return (
-    <button type="button" className="cal-btn">
-      {label}
-    </button>
-  );
-}
+export default function Calculator() {
+  const [displayValue, setDisplayValue] = useState('0');
+  const [storedValue, setStoredValue] = useState(null);
+  const [operator, setOperator] = useState(null);
 
-Button.propTypes = { label: PropTypes.string.isRequired };
+  const handleDigitClick = (digit) => {
+    if (displayValue === '0') {
+      setDisplayValue(digit);
+    } else {
+      setDisplayValue((prevDisplay) => prevDisplay + digit);
+    }
+  };
+
+  const performCalculation = () => {
+    const currentValue = parseFloat(displayValue);
+
+    if (operator === '+') {
+      setDisplayValue(storedValue + currentValue);
+    } else if (operator === '-') {
+      setDisplayValue(storedValue - currentValue);
+    } else if (operator === '*') {
+      setDisplayValue(storedValue * currentValue);
+    } else if (operator === '/') {
+      setDisplayValue(storedValue / currentValue);
+    }
+
+    setStoredValue(null);
+    setOperator(null);
+  };
+
+  const handleOperatorClick = (op) => {
+    if (operator && storedValue !== null) {
+      performCalculation();
```

Comment:
You aren't supposed to be doing the calculator logic all by yourself, consider copying these files ([one](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js) and [two](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js)) as stated in the project requirements into your project for usage. This is to help you focus on the main react stuff for this milestone.